app/models/deploy.rb

```diff
@@ -110,7 +110,7 @@ def self.finished_naturally
   end
 
   def self.prior_to(deploy)
-    deploy.persisted? ? where("#{table_name}.id < ?", deploy.id) : self
+    deploy.persisted? ? where("#{table_name}.id < ?", deploy.id) : all
```

Comment:
You sure all is deprecated? I think it was a bug in 4.0 saying it's
deprecated
On 12/03/2015 9:28 am, "Michael Grosser" <notifications@github.com> wrote:

> In app/models/deploy.rb
> <https://github.com/zendesk/samson/pull/289#discussion_r26262632>:
>
> > @@ -110,7 +110,7 @@ def self.finished_naturally
> >    end
> >
> >    def self.prior_to(deploy)
> > -    deploy.persisted? ? where("#{table_name}.id < ?", deploy.id) : self
> > +    deploy.persisted? ? where("#{table_name}.id < ?", deploy.id) : all
>
> all is deprecated ... maybe .where() or .where('1=1') ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/289/files#r26262632>.
>